openapi/integrator/token-requestor/partner.yaml

```diff
@@ -176,16 +176,39 @@ components:
         * Accepted - Enrolment was accepted.
         * Rejected - Enrolment was rejected.
         * Failed - Enrolment failed.
-        * InvalidFundingPan - No eligible card found.
-        * CardExpired - Card has expired.
-        * CardNotActivated - Card is not activated.
       enum:
         - Accepted
         - Rejected
         - Failed
-        - InvalidFundingPan
-        - CardExpired
-        - CardNotActivated
+
+    ErrorCode:
+      type: string
+      example: Accepted
+      description: |
+        Code specifying the result of the operation:
+        * ENROLMENT_OPERATION_FAILED -
```

Comment:
Why duplicating enum values in the description? Enum values itself are self-describing. Do thoses ` -` at the end of some of them have a special meaning?

Reply:
Resolved.